src/main/java/com/github/ibm/mapepire/requests/PreparedExecute.java

```diff
@@ -59,8 +56,7 @@ protected void go() throws Exception {
         } else {
             addReplyData("data", new LinkedList<Object>());
             addReplyData("has_results", false);
-            addReplyData("update_count", stmt.getLargeUpdateCount());
-            m_rs.close();
```

Comment:
Don't we still need to call this `close()`?

Reply:
That close would actually throw a null pointer exception because the result set is null in that else block. That's why I removed it.

Reply:
@jonnyz32 fyi, I also caught this the `m_rs` mistake and fixed as part of #44

Rebasing might be fun since we'll have some merge conflicts with my PR, so just a heads-up